Feature/io reading from console into text file

krisVVS opened the pull request.

krisVVS committed:
second task almost completet. have to add unit tests and javadoc

krisVVS committed:
Update ci.yml

Commit:
complete 2nd task of module 6

# Conflicts:
#	Input-Output/src/test/java/com/onto/javacourse/inputoutput/consolereader/ConsoleReaderTest.java

Commit:
complete 2nd task of module 6

# Conflicts:
#	Input-Output/src/test/java/com/onto/javacourse/inputoutput/consolereader/ConsoleReaderTest.java

Commit:
removed some useless files

Commit:
fixed the exception handling

ivorusev commented on Input-Output/src/main/java/com/onto/javacourse/inputoutput/fromconsoletofile/FromConsoleToFile.java:
```diff
@@ -0,0 +1,51 @@
+      String line;
+
+      System.out.println("Enter text to write to file. Type '.' on a new line to stop.");
+      while ((line = reader.readLine()) != null && !line.equals(".")) {
```
".".equals(line)

ivorusev commented on Input-Output/src/test/java/com/onto/javacourse/inputoutput/fromconsoletofile/FromConsoleToFileTest.java:
```diff
@@ -0,0 +1,76 @@
+    File tempFile = File.createTempFile("test_empty", ".txt");
+    tempFile.deleteOnExit();
+
+    String simulatedInput = "test_empty.txt\n.\n";
```
System.lineSeparator()

Commit:
fixed the tests

# Conflicts:
#	Input-Output/src/test/java/com/onto/javacourse/inputoutput/consolereader/ConsoleReaderTest.java

Commit:
fixed messages

Commit:
fixed the messages

Commit:
fixed new error

Commit:
again removed files

Commit:
checkstyle edit

Commit:
again trying to resolve conflict

Commit:
try again
